### PR TITLE
Render all texts on component example

### PR DESCRIPTION
**Pull request by rvsia**
- inactive are hidden
- to support algolia
- there is only one header in the documentation of each component's implementation

### CODE REVIEW

File: packages/react-renderer-demo/src/app/src/components/docsearch.js [hunk 1]
```diff
@@ -52,6 +52,7 @@ const DocSearch = () => {
       indexName: 'data-driven-forms',
       inputSelector: '#data-driven-forms-search',
       handleSelected: (input, event, suggestion) => {
+        console.log(event.target.value, suggestion);
```
Comment (Hyperkid123) [inline packages/react-renderer-demo/src/app/src/components/docsearch.js]: console log

Comment (rvsia) [reply packages/react-renderer-demo/src/app/src/components/docsearch.js]: :man_facepalming:

**Comment by rvsia** [comment]
@Hyperkid123 fixed

**Pull request merged**